### DISCUSSION TITLE
Do I need to know SQL to use Prisma

### DISCUSSION THREAD

**Discussion by muco-rolle** [prisma/prisma#6697] [Q&A] [answered]
Hello everyone, as someone who learned Nodejs with MongoDB and it seems like the support of MongoDB is not ready yet I would like to understand how much knowledge I will need to start using Prisma and what resources can I  use to learn SQL

**Comment by KnowYourselves** [comment]
Hello @muco-rolle!

I started learning Prisma the other day and I would say you need little to no knowledge of SQL. The only thing you kinda need to understand is how Prisma communicates with your database (the Prisma schema, migrations, etc) and that's it!

That's the nice thing about ORMs like Prisma, the idea is to use the ORM instead of doing SQL queries.

**Comment by pr0ph0z** [comment]
Not the advanced one but if you have used Node.js and MongoDB using Mongoose, Prisma might be suitable for you because Prisma utilizes Javascript Object for its query

**Comment by 2color** [comment, marked as answer]
Hey @muco-rolle 👋,

Knowledge of SQL is not strictly required to use Prisma. However, understanding how data modeling works in relational databases would be greatly useful. 

I highly recommend reading the [Data modeling](https://www.prisma.io/dataguide/datamodeling) guide on Prisma's Data Guide. 

If you want to learn SQL, check out this Zine by Julian Evans: https://wizardzines.com/zines/sql/. It will make SQL a lot easier to understand. 

Happy learning!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
